eva/utils/profiler.py

```diff
@@ -42,10 +54,22 @@ def run(self):
         Param: None
 
         Returns: List[Metrics]
-        """
-        # TODO: Implement the actual logic
-        # Use self._classobj's methods to run for various batch sizes
+        """        
+
+        #NOTE This wont work if input_format is -1. Need to implement for that case.
```

Comment:
Seems like fastrcnn gives -1. What do we in that case?

Ref: https://github.com/aravindhank11/eva-cost-base-optimizer/blob/master/eva/udfs/fastrcnn_object_detector.py#L59-L61

Also is `-1` any arbitrary size?

Reply:
Yea we'll need to check how to handle for -1 case. From what I know, we can't really get the input vector shape using just the model object.

Reply:
Yes. I think -1 should be any size in this case